minimum-constraints.txt

```diff
@@ -311,7 +312,8 @@ jupyterlab-widgets==1.0.2; python_version == '3.6'
 jupyterlab-widgets==3.0.3; python_version >= '3.7'
 
 jupyter-server>=1.24.0; python_version == '3.7'
-jupyter-server>=2.4.0; python_version >= '3.8'
+# Safety issue with noteboook.
+jupyter-server==2.7.2; python_version >= '3.8'
```

Comment:
notebook 7.0.6 requires jupyter-server >=2.4.0,<3.
What was the reason to increase its minimum version to 2.7.2?
I suggest to increase it only when there is a reason.

Reply:
I did this because specifically in the vulnerability it states:

    Users are advised to upgrade jupyter-server to version
    2.7.2 or newer, which includes a fix for a redirect vulnerability.

That is really a questionable change and probably should never have been included only as a comment.  It is either a package security issue and deserves a Vulnerability issue or not.  

**DISCUSSION**

Reply:
I suggest to include it, and the current version already does. DONE.